consumers/campaigner/main.go

```diff
@@ -110,8 +110,9 @@ func (h *MessageHandler) HandleMessage(m *nsq.Message) error {
 					"campaign_id": campaign.ID,
 					"status":      campaign.Status,
 				}).Error("unable to update campaign")
-			return nil
+			return err
```

Comment:
@gudgl we should not return err here since the campaign status is already set to sending.

Reply:
@Dzalevski agreed